include/oneapi/internal/math/blas_executor.h

```diff
@@ -24,13 +24,7 @@
 //--
 */
 
-#ifdef ONEAPI_DAAL_USE_MKL_GPU_FUNC
-    #if !(defined(__clang__))
-        #undef ONEAPI_DAAL_USE_MKL_GPU_FUNC
-    #endif
-#endif
```

Comment:
Why did you remove this condition? MKL is expected to work only with clang. I would replace it with check for Intel DPC++ compiler not only for clang

Reply:
Have a look at [this line](https://github.com/intel/daal/blob/master/include/services/daal_defines.h#L102), it checks for Intel DPC++ compiler `__SYCL_COMPILER_VERSION`

Reply:
fixed